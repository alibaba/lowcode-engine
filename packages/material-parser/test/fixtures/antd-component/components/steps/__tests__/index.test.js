import Steps from '..';
import mountTest from '../../../tests/shared/mountTest';
import rtlTest from '../../../tests/shared/rtlTest';

describe('Steps', () => {
  mountTest(Steps);
  rtlTest(Steps);
});
