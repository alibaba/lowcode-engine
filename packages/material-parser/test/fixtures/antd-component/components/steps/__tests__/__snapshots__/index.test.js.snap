// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Steps rtl render component should be rendered correctly in RTL direction 1`] = `
<div
  class="ant-steps ant-steps-horizontal ant-steps-rtl ant-steps-label-horizontal"
/>
`;
